app/angular/src/client/preview/angular/NgComponentAnalyzer.test.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  Component,
+  ComponentFactory,
+  ComponentFactoryResolver,
+  EventEmitter,
+  Input,
+  Output,
+  Type,
+} from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/testing';
+
+import { getComponentInputsOutputs } from './NgComponentAnalyzer';
+
+describe('getComponentInputsOutputs', () => {
```

Comment:
Awesome test suite, it will make us much more confident about everything related to component properties detection 💪🏻